include/roaring/roaring.h

```diff
@@ -815,6 +815,15 @@ bool roaring_bitmap_select(const roaring_bitmap_t *r, uint32_t rank,
  */
 uint64_t roaring_bitmap_rank(const roaring_bitmap_t *r, uint32_t x);
 
+/**
+ * roaring_bitmap_rank_many is an `Bulk` version of `roaring_bitmap_rank`
+ * it puts rank value of each element in `[begin .. end)` to `ans[]`
+ *
+ * the values in `[begin .. end)` must be sorted in Ascending order;
+ * the `ans` must have enough size.
```

Comment:
May be we can be more precise and state what is meant by _size_. It is `size_t length = (end-begin)`. Correct?

Reply:
ok. i borrowed a doc from `roaring_bitmap_to_uint32_array()`. 
```
 *     ans = malloc((end-begin) * sizeof(uint64_t));